docs/data-sources/dc_virtual_interfaces.md

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Direct Connect (DC)"
+---
+
+# huaweicloud_dc_virtual_interfaces
+
+Use this data source to get the list of DC virtual interfaces.
+
+## Example Usage
+
+```hcl
+variable "direct_connect_id" {}
+
+data "huaweicloud_dc_virtual_interfaces" "test" {
+  direct_connect_id = var.direct_connect_id
```

Comment:
`direct_connect_id` is not defined in schema

Reply:
done